csm/core/agent/api.py

```diff
@@ -369,7 +378,7 @@ def run(port: int, https_conf: ConfSection, debug_conf: DebugConf):
             ssl_context = None
 
         web.run_app(CsmRestApi._app, port=port, ssl_context=ssl_context,
-                    access_log=Log.logger, access_log_format=const.REST_ACCESS_FORMAT)
+                    access_log=None, access_log_format=const.REST_ACCESS_FORMAT)
```

Comment:
`access_log_format` is an optional parameter and, if we are not using it---which seems to be the case now that `access_log` is `None`, it can be removed, along with `const.REST_ACCESS_FORMAT` .

Reply:
I have kept in case for debugging if want to turn it on.  Yes it is not used now.

Reply:
@tb-s Let me know if we really want to remove it

Reply:
@navalkp I would remove it.

Reply:
Removed it.